tests/integration_tests/functional/test_mmds.py

```diff
@@ -649,7 +647,7 @@ def test_mmds_limit_scenario(test_microvm_with_api, network_config, version):
 
 
 @pytest.mark.parametrize("version", MMDS_VERSIONS)
```

Comment:
Is it worth it making a fixture of this as well?

Reply:
It's only used in this test, so I think not much.